Highway.h

```diff
@@ -2,6 +2,8 @@
 
 #include <vector>
 #include <algorithm>
+#include "Vehicle.h"
+#include "HighwayPatrol.h"
```

Comment:
Don't include what you don't use.  Verify if you need all of these in the header file or if you can move some to the cpp file.
- [ ] revise